tasks/gen/main.go

```diff
@@ -0,0 +1,128 @@
+package main
```

Comment:
it would be a bit easier to just have this in the tasks package as a `gen.go` with `// +build ignore`, since you avoid the directory. another big plus is that the gen package doesn't pollute the module in `go list ./...`.